homeassistant/components/openhome/media_player.py

```diff
@@ -20,35 +21,45 @@
     SUPPORT_VOLUME_STEP,
 )
 from homeassistant.const import STATE_IDLE, STATE_OFF, STATE_PAUSED, STATE_PLAYING
+from homeassistant.helpers import config_validation as cv, entity_platform
+
+from .const import ATTR_PIN_INDEX, DATA_OPENHOME, SERVICE_INVOKE_PIN
 
 SUPPORT_OPENHOME = SUPPORT_SELECT_SOURCE | SUPPORT_TURN_OFF | SUPPORT_TURN_ON
 
 _LOGGER = logging.getLogger(__name__)
 
-DEVICES = []
-
 
-def setup_platform(hass, config, add_entities, discovery_info=None):
+async def async_setup_platform(hass, config, async_add_entities, discovery_info=None):
     """Set up the Openhome platform."""
 
     if not discovery_info:
-        return True
+        return
+
+    openhome_data = hass.data.setdefault(DATA_OPENHOME, [])
```

Comment:
Let's make this a set instead. Please use `set()` as default value instead of `[]`.